packages/arb-avm-cpp/cavm/carbstorage.h

```diff
@@ -32,7 +33,10 @@ CArbStorage* createArbStorage(const char* db_path,
                               int32_t lru_cache_size,
                               int32_t debug,
                               int32_t save_rocksdb_interval,
-                              const char* save_rocksdb_path);
+                              const char* save_rocksdb_path,
+                              int64_t profile_reorg_to,
+                              int64_t profile_run_until,
+                              int64_t profile_load_count);
```

Comment:
In #1600 this is changed into a struct. Likely we'll want to merge this one first but something to be aware of.

Reply:
copied the struct into this PR to reduce conflicts